lib/datasource/terraform-provider/index.spec.ts

```diff
@@ -151,4 +154,135 @@ describe(getName(), () => {
       expect(httpMock.getTrace()).toMatchSnapshot();
     });
   });
+  describe('getBuilds', () => {
+    beforeEach(() => {
+      jest.clearAllMocks();
+    });
```

Comment:
```suggestion
```

Not needed.

Reply:
Why resolved without change or comment?

Reply:
I have mistakenly deleted that from the `getReleases` tests, but as it seems not to be needed anymore I will keep it deleted.